src/bin/assets/catppuccin-highlight.css

```diff
@@ -1,359 +1,571 @@
+  color: #8caaee;
 }
 .latte code .hljs-tag {
-  color: #d20f39;
+  color: #a5adce;
 }
 .latte code .hljs-name {
-  color: #dd7878;
-}
-.latte pre .hljs {
-  background: #e6e9ef !important;
-}
-.latte pre .hljs-params {
-  color: #d20f39 !important;
-}
-.latte pre .hljs-built_in,
-.latte pre .hljs-selector-tag,
-.latte pre .hljs-section,
-.latte pre .hljs-link {
-  color: #209fb5 !important;
-}
-.latte pre .hljs-keyword {
-  color: #8839ef !important;
-}
-.latte pre .hljs,
-.latte pre .hljs-subst {
-  color: #6c6f85 !important;
-}
-.latte pre .hljs-title {
-  color: #1e66f5 !important;
-}
-.latte pre .hljs-attr,
-.latte pre .hljs-meta-keyword {
-  color: #40a02b !important;
-}
-.latte pre .hljs-type {
-  color: #1e66f5 !important;
-}
-.latte pre .hljs-string {
-  color: #40a02b !important;
-}
-.latte pre .hljs-tag {
-  color: #d20f39 !important;
-}
-.latte pre .hljs-meta,
-.latte pre .hljs-name,
-.latte pre .hljs-symbol,
-.latte pre .hljs-bullet,
-.latte pre .hljs-addition,
-.latte pre .hljs-variable,
-.latte pre .hljs-template-tag,
-.latte pre .hljs-template-variable {
-  color: #dd7878 !important;
-}
-.latte pre .hljs-addition {
-  background-color: #e6e9ef !important;
-  color: #40a02b !important;
-}
-.latte pre .hljs-deletion {
-  background-color: #e6e9ef !important;
-  color: #d20f39 !important;
-}
-.latte pre .hljs-comment,
-.latte pre .hljs-quote {
-  color: #acb0be !important;
-}
-.latte pre .hljs-keyword,
-.latte pre .hljs-selector-tag,
-.latte pre .hljs-literal,
-.latte pre .hljs-title,
-.latte pre .hljs-section,
-.latte pre .hljs-doctag,
-.latte pre .hljs-type,
-.latte pre .hljs-name,
-.latte pre .hljs-strong {
-  font-weight: bold !important;
-}
-.latte pre .hljs-literal,
-.latte pre .hljs-number {
-  color: #fe640b !important;
-}
-.latte pre .hljs-emphasis {
-  font-style: italic !important;
+  color: #ca9ee6;
+}
+.latte code .hljs-attr {
+  color: #8caaee;
+}
+.latte code .hljs-attribute {
+  color: #a6d189;
+}
+.latte code .hljs-bullet {
+  color: #81c8be;
+}
+.latte code .hljs-code {
+  color: #a6d189;
+}
+.latte code .hljs-emphasis {
+  color: #e78284;
+  font-style: italic;
+}
+.latte code .hljs-strong {
+  color: #e78284;
+  font-weight: bold;
+}
+.latte code .hljs-formula {
+  color: #81c8be;
+}
+.latte code .hljs-link {
+  color: #85c1dc;
+  font-style: italic;
+}
+.latte code .hljs-quote {
+  color: #a6d189;
+  font-style: italic;
+}
+.latte code .hljs-selector-tag {
+  color: #e5c890;
+}
+.latte code .hljs-selector-id {
+  color: #8caaee;
+}
+.latte code .hljs-selector-class {
+  color: #81c8be;
+}
+.latte code .hljs-selector-attr {
+  color: #ca9ee6;
+}
+.latte code .hljs-selector-pseudo {
+  color: #81c8be;
+}
+.latte code .hljs-template-tag {
+  color: #eebebe;
+}
+.latte code .hljs-template-variable {
+  color: #eebebe;
+}
+.latte code .hljs-diff-addition {
+  color: #a6d189;
+  background: rgba(var(--ctp-green), 15%);
+}
+.latte code .hljs-diff-deletion {
+  color: #e78284;
+  background: rgba(var(--ctp-red), 15%);
```

Comment:
Just one more note on this whole file, it would be better if we keep this out of VCS in the future, since it's generated.

Reply:
Sounds good, I'll get around to making a PR to removing it from VCS and modifying the CI workflow to generate this file too